### PR TITLE
changes needed for django 1.9 upgrade

### PR DESCRIPTION
**Pull request by mmchugh**
Changed up fields and methods that were removed, and switched from south to django migrations

### CODE REVIEW

**Comment by toastdriven** [comment]
:wind_chime:

File: mithril/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # (c) 2012 Urban Airship and Contributors
 
-__version__ = (0, 1, 0)
+__version__ = (0, 2, 0)
```
Comment (coddingtonbear) [inline mithril/__init__.py]: This looks like it's technically backwards incompatible (changing from south to django's migrations, dropping python 2.6 support); assuming anybody's using this outside, we should probably try to be good citizens and do a major version bump, no?

**Comment by coddingtonbear** [comment]
Other than that --

:boom:
:bomb:

**Comment by cowalla** [comment]
:cactus:

**Pull request merged**